src/harness/reference_models/interference/interference.py

```diff
@@ -413,57 +394,73 @@ def computeInterferenceFssCochannel(cbsd_grant, constraint, fss_info, max_eirp):
 def getFssMaskLoss(cbsd_grant, constraint):
   """Gets the FSS mask loss for a FSS blocking protection constraint."""
   # Get 50MHz offset below the lower edge of the FSS earth station
-  offset = constraint.low_frequency - 50.e6
-
+  offset = constraint.high_frequency - 50*MHZ
   # Get CBSD grant frequency range
   cbsd_freq_range = cbsd_grant.high_frequency - cbsd_grant.low_frequency
+  # Define the initial attenuation values to 0
+  fss_mask_attenuation = 0
+  fss_mask_attenuation_seg1 = 0
+  fss_mask_attenuation_seg2 = 0
+  # If the grant overlap frequency lies between fp1-50 MHz to fp1, where fp1 is the lower edge of the FSS's passband
+  # Segment 1 calculation is applied
+  if ( offset <= cbsd_grant.low_frequency and
+         constraint.high_frequency >= cbsd_grant.high_frequency):
+    # Calculate the total of 1MHZ frequencies present in the grant frequency range
+    num_one_mhz_bin_seg1 = cbsd_freq_range/int(MHZ)
```

Comment:
I guess you want the integer part, then use:
int(cbsd_freq_range/MHZ), and not the current line

Otherwise if cbsd_freq_range, it will still do a float point division giving back a floating point

Reply:
thanks.